README.md

```diff
@@ -78,54 +61,71 @@ or
 $ conda install -c conda-forge boost
 ```
 
+## Script to build cuDF from source
+
 ### Build from Source
 
 To install cuDF from source, ensure the dependencies are met and follow the steps below:
 
-1. Clone the repository
+- Clone the repository recursively
 ```bash
 git clone --recurse-submodules https://github.com/rapidsai/cudf.git
 cd cudf
 ```
-2. Create the conda development environment `cudf` as detailed above
-3. Build and install `libcudf`
+- Create the conda development environment `cudf_dev`
 ```bash
-$ cd /path/to/cudf/cpp                              # navigate to C/C++ CUDA source root directory
+# create the conda environment (assuming in base `cudf` directory)
+conda env create --name cudf_dev --file conda/environments/dev_py35.yml
+# activate the environment
+source activate cudf_dev
+```
+
+- Build and install `libcudf`. CMake depends on the `nvcc` executable being on your path or defined in `$CUDACXX`.
+```bash
+$ cd cpp                                            # navigate to C/C++ CUDA source root directory
 $ mkdir build                                       # make a build directory
 $ cd build                                          # enter the build directory
 $ cmake .. -DCMAKE_INSTALL_PREFIX=/install/path     # configure cmake ... use $CONDA_PREFIX if you're using Anaconda
 $ make -j                                           # compile the libraries librmm.so, libcudf.so ... '-j' will start a parallel job using the number of physical cores available on your system
 $ make install                                      # install the libraries librmm.so, libcudf.so to '/install/path'
 ```
-To run tests (Optional):
 
+- To run tests (Optional):
 ```bash
 $ make test
 ```
 
-Build and install cffi bindings:
+- Build and install cffi bindings:
 ```bash
 $ make python_cffi                                  # build CFFI bindings for librmm.so, libcudf.so
 $ make install_python                               # build & install CFFI python bindings. Depends on cffi package from PyPi or Conda
 $ py.test -v                                        # optional, run python tests on low-level python bindings
 ```
 
-4. Build the `cudf` python package, in the `python` folder:
+- Build the `cudf` python package, in the `python` folder:
```

Comment:
I think the number 4 should be left. This is a separate build step from libcudf.
```suggestion
- 4. Build the `cudf` python package, in the `python` folder:
```